RecoEcal/EgammaClusterProducers/BuildFile.xml

```diff
@@ -4,6 +4,8 @@
 <use name="DataFormats/EcalDigi"/>
 <use name="DataFormats/EcalRecHit"/>
 <use name="DataFormats/EgammaReco"/>
+<use name="DataFormats/RPCRecHit"/>
+<use name="DataFormats/GEMRecHit"/>
```

Comment:
Why are these changes needed?

Reply:
@thomreis  TrackAssociatorParameters are loaded in https://github.com/cms-sw/cmssw/blob/b0b0e732c03ad78d999359313705e18a767c6ddb/RecoEcal/EgammaClusterProducers/src/InterestingTrackEcalDetIdProducer.cc#L84 and there were some new parameters added to the TrackAssociatorParameters class. Without these lines I got a break segmentation fault when I was first implementing it.